docs/ossm/ambient/waypoint.md

```diff
@@ -0,0 +1,233 @@
+* **Enhanced Security:** Provides a secure, zero-trust network foundation with mTLS by default for all meshed workloads.
+
+**Trade-offs:**
+
+* Ambient mode is a newer architecture and may have different operational considerations compared to the traditional sidecar model.
+* L7 features require the deployment of `waypoint` proxies, which add a small amount of overhead for the services that utilize them.
+
+---
+
+## 2. Pre-requisites to Using Waypoint Ambient Mode with OSSM 3
+
+Before installing Istio's ambient mode with OpenShift Service Mesh, ensure the following prerequisites are met:
+
+* **OpenShift Container Platform 4.19+:** This version of OpenShift is required for supported Kubernetes Gateway API CRDs, which are essential for ambient mode functionalities.
+* **OpenShift Service Mesh 3.1.0+ operator is installed:** Ensure that the OSSM operator version 3.1.0 or later is installed on your OpenShift cluster.
+* **Istio deployed in Ambient Mode:** Refer to OSSM Ambient [initial doc](README.md).
+
+**Pre-existing Service Mesh Installations:**
+
+While the use of properly defined discovery selectors will allow a service mesh to be deployed in ambient mode alongside a service mesh in sidecar mode, this is not a scenario we have thoroughly validated. To avoid potential conflicts, as a technology preview feature, Istio's ambient mode should only be installed on clusters without a pre-existing OpenShift Service Mesh installation.
+
+**Note:**: Istio's ambient mode is completely incompatible with clusters containing the OpenShift Service Mesh 2.6 or earlier versions of the operator and they should not be used together.
+
+---
+
+## 3. Waypoint Proxies
+
+Ambient mesh splits Istio's functionality into two distinct layers, a secure overlay layer 4 (L4) and a Layer 7 (L7). The waypoint proxy is an optional component that is Envoy-based and handles L7 processing for workloads it manages. It acts as a gateway to a resource (a namespace, service or pod). Waypoint proxies are installed, upgraded and scaled independently from applications. They can be configured using the Kubernetes [Gateway API](https://gateway-api.sigs.k8s.io/).
+
+### 3.1 Core features
+
+If your applications require any of the following L7 mesh functions, you will need to use a waypoint proxy in ambient mode:
+
+- **Traffic management:** HTTP routing & load balancing, circuit breaking, rate limiting, fault injection, retries and timeouts
+- **Security:** Rich authorization policies based on L7 primitives such as request type or HTTP header
+- **Observability**: HTTP metrics, access logging and tracing
+
+### 3.2 Prerequisites
+
+Waypoint proxies are deployed using Kubernetes Gateway resources.  
+**Note:** As of OpenShift 4.17, the Kubernetes Gateway API CRDs are not available by default and must be installed to be used. This can be done with the following command:
+
+```bash
+kubectl get crd gateways.gateway.networking.k8s.io &> /dev/null || \
```

Comment:
Should use an `oc` command, but also add a comment that these are community Kubernetes CRDs, which are not supported by Red Hat.